index.html

```diff
@@ -33,40 +42,63 @@ <h2 class="sub-t">I’am a Software Developer</h2>
+                    <ul class="languages">
+                        <li class="g-item">html</li>
+                        <li class="g-item">css</li>
+                        <li class="g-item">bootstrap</li>
+                        <li class="g-item">ruby</li>
+                    </ul>
+                    <button type="button" class="but1" >See project</button>
+                </div>
+            <div class="title-med">
+                <div class="img-med"><img src="./images/Img Placeholder1.png"></div>
+                <div class="img-medm"><img src="./images/Img Placeholder.png"></div>
+                <div  class="title-tree">
+                    <h2 class="multi">Multi-Post Stories</h2>
+                    <p class="daily">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <ul class="languages">
+                        <li class="g-item">html</li>
+                        <li class="g-item">css</li>
+                        <li class="g-item">bootstrap</li>
+                        <li class="g-item">ruby</li>
+                    </ul>
+                    <button type="button" class="but2" >See project</button>
+                </div>
             </div>
+            
         </section>
         <section>
 
             <div id="frams">
-                <div class="grid-item">
-                    <h2 class="prof"> Profesional Art Printing Data</h2>
-                    <p class="daily1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-                    <ul class="languagess">
-                        <li class="g-item1">html</li>
-                        <li class="g-item1">css</li>
-                        <li class="g-item1">bootstrap</li>
-                        <li class="g-item1">ruby</li>
-                    </ul>
+                <div class="grid-item1">
+                    <div class="grid-medd">
+                        <h2 class="prof"> Profesional Art Printing Data</h2>
+                        <h2 class="proff">Data DashboardHealthcare</h2>
+                        <p class="daily1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul class="languagess">
+                            <li class="g-item1">html</li>
+                            <li class="g-item1">css</li>
+                            <li class="g-item1">bootstrap</li>
+                            <li class="g-item1">ruby</li>
+                        </ul>
+                    </div>
                     
```

Comment:
- [ ] As mentioned by the previous reviewer that when the project is hovered on with the mouse the description and dark opacity of the background image disappear and a See Project button appears at the bottom of the image and takes the whole image width mentioned button should take you to the Live Demo of the project or to the GitHub repository. Please implement this and then you are good to go.